cpu/stm32l1/periph/i2c.c

```diff
@@ -116,9 +116,9 @@ int i2c_init_master(i2c_t dev, i2c_speed_t speed)
         /* disable peripheral */
         i2c->CR1 &= ~I2C_CR1_PE;
         /* toggle both pins to reset analog filter */
-        _toggle_pins(port, pin_scl, pin_sda);
+        _toggle_pins(dev);
```

Comment:
I mean: The toggling is **not** I2C specific. I suggested this is a copy-paste "error" from other MCUs. So if you don't find a hint in the corresponding errata sheet (for **your**) MCU it is possible that the toggling is not needed. Just do the test if everything goes as expected **without** it. In that case you can remove the whole shit